api/controllers/userController.js

```diff
@@ -0,0 +1,15 @@
+const userService = require("../services/userService");
+const { catchAsync } = require("../middlewares/error");
+const { detectError } = require("../utils/detectError");
+
+const kakaoLogin = catchAsync(async (req, res) => {
+  const kakaoToken = req.headers.authorization;
+
+  if (!kakaoToken) detectError("NEED_ACCESS_TOKEN", 401);
```

Comment:
카카오 로그인을 위해 토큰을 받는 과정으로 보입니다.

다만 클라이언트가 전달하는 카카오 토큰이 없을 때 Access token이 없다는 에러 메시지를 반환할 경우, 인가 과정에서 accessToken을 전달하는 클라이언트에게 다소 애매한 에러메시지가 될 수 있을 것 같습니다.

```suggestion
  if (!kakaoToken) detectError("NEED_KAKAO_TOKEN", 401);
```